DSL.Resql/get_corpora.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM list_corpora_tasks_paginated(:where_condition, :sort_condition, :page, :page_size);
```

Comment:
Avoid `select *` in any case and be explicit on what you are requesting.

Reply:
Done